pkg/app/piped/platformprovider/lambda/client.go

```diff
@@ -295,7 +295,23 @@ func (c *client) updateFunctionConfiguration(ctx context.Context, fm FunctionMan
 	if !updateFunctionConfigurationSucceed {
 		return fmt.Errorf("failed to update configuration for Lambda function %s: %w", fm.Spec.Name, err)
 	}
-	return nil
+
+	// Wait until function updated successfully.
+	input := &lambda.GetFunctionInput{
+		FunctionName: aws.String(fm.Spec.Name),
+	}
+	_, err = retry.Do(ctx, func() (any, error) {
```

Comment:
Sorry, I just noticed that we reused the previous retry object here. I think we should recreate a new one instead since the same object may contain previously updated backoff internal, thus the number of retry times is not as expected.

```suggestion
	retry = backoff.NewRetry(RequestRetryTime, backoff.NewConstant(RetryIntervalDuration))
	_, err = retry.Do(ctx, func() (any, error) {
```

Reply:
@khanhtc1202 
I fixed it.